src/mobile/package.json

```diff
@@ -15,19 +15,19 @@
     "@react-navigation/native": "6.0.7",
     "@react-navigation/native-stack": "6.3.0",
     "expo": "44.0.6",
-    "expo-status-bar": "1.2.0",
-    "expo-updates": "0.11.6",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.67.3",
+    "expo-status-bar": "~1.2.0",
+    "expo-updates": "~0.11.6",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
     "react-native-elements": "3.4.2",
     "react-native-safe-area-context": "3.3.2",
-    "react-native-screens": "3.10.2",
-    "react-native-web": "0.17.6"
+    "react-native-screens": "~3.10.1",
+    "react-native-web": "0.17.1"
   },
   "devDependencies": {
     "@senecacdot/eslint-config-telescope": "1.0.0",
-    "@babel/core": "7.17.5"
+    "@babel/core": "^7.12.9"
```

Comment:
Do you change version because of this issue #3107?

Reply:
Same goes with other dependencies

Reply:
yeah so basically when I try to run the project I have to use `expo doctor` and `expo update` follow #3170

Reply:
else it would break since the version is not compatible with the `react-native` virtual machine, that what I understand about it

Reply:
Aah got ya!